tests/openssl/enc/CMakeLists.txt

```diff
@@ -29,13 +29,19 @@ add_enclave_library(
   # Ensure that openssl_t.h is generated.
   openssl_t.h)
 
+enclave_compile_definitions(openssl-support PRIVATE __STDC_NO_ATOMICS__)
```

Comment:
I think all the changes in tests/openssl/enc/CMakeLists.txt might be from merge conflicts, I don't think this file should be changed at all